ui/packages/components/src/AppRoot/AppRoot.tsx

```diff
@@ -15,7 +13,7 @@ export function AppRoot({
   return (
     <html
       lang="en"
-      className={`${devServer ? interTight.variable : ''} ${devServer ? robotoMono : ''} ${
+      className={`${devServer ? interTight.variable : ''} ${devServer ? robotoMono.variable : ''} ${
```

Comment:
Why is the Dev Server not changing to the new font as well?

Reply:
Because there was push back that we shouldn't be doing a new font at all....because the change is disruptive....I put it in in the least disruptive was possible for me to build the new dashboard IA with it. 

As I said in the original PR, I'm happy to add it to the dev server as well if people are ok with the disruption. IMHO it looks fine there as well.

Reply:
The goal of the design system is to keep the consistency for users, so it builds trust in our product and the brand. By changing the font, we only achieve it if we do it in both products, otherwise we are introducing more inconsistency. Since there's no extra work in making it happen in the dev server, and we've seen it looks good, let's do it everywhere!

I'll approve this PR since there's great work here

Reply:
Yup, will do.